doc/rules-reference.md

```diff
@@ -51,8 +52,14 @@ for each violated rule.
 
 ### `allowed`
 A list of rules that describe dependencies that are _allowed_. dependency-cruiser
-will emit the warning message 'not-in-allowed' for each dependency that does not
-satisfy at least one of them.
+will emit a 'not-in-allowed' message for each dependency that does not
+satisfy at least one of them. The severity of the message is _warning_ by
+default, but you can override it with `
```

Comment:
something is missing at the end of this line

Reply:
good catch!